llm_api_hub/server/app.py

```
@@ -364,10 +409,60 @@ def models(request: Request) -> Dict[str, Any]:
+        return {
+            "id": response["id"],
+            "type": "message",
+            "role": "assistant",
+            "content": [{"type": "text", "text": content}],
+            "model": response["model"],
+            "usage": response.get("usage", {"input_tokens": 0, "output_tokens": 0})
+        }
+    return response
+
+@app.post("/v1/models/{model_name}:generateContent")
+def google_generate_content(model_name: str, request_data: Dict[str, Any], raw_request: Request) -> Any:
```

Pylint (4.0.6)

[convention] Line too long (104/100)

(C0301)